freqtrade/strategy/interface.py

```diff
@@ -27,23 +27,23 @@
 
 
 logger = logging.getLogger(__name__)
-CUSTOM_SELL_MAX_LENGTH = 64
+CUSTOM_EXIT_MAX_LENGTH = 64
 
 
-class SellCheckTuple:
+class ExitCheckTuple:
```

Comment:
Doublecheck if this is a public interface - in which case we'll want to cleanly deprecate and have a grace period.